### PR TITLE
request to github is successful. then we send the json as response to client

### PR DESCRIPTION
**Pull request by izelnakri**
… web server

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,9 +10,21 @@ app.get('/', (request, response) => {
...
+app.get('/profile', (req, res) => {
+  request({
+    method: 'GET',
+    url: 'https://api.github.com/users/izelnakri',
+    headers: {
+      'User-Agent': 'Izels awesome JS Code'
+    }
+  }, (error, response, body) => {
+    if (!error) {
+      res.type('application/json');
```
Comment (izelnakri) [inline index.js]: Anybody knows why we have this line?

Comment (isisvanderplas) [reply index.js]: if it's truthy, run the app

Comment (Zahra120) [reply index.js]: Somehow!

Comment (Saidac) [inline index.js]: if not error give  response with given pathway

**Review by isisvanderplas — COMMENTED** [review]
if it's truthy, run the app

File: index.js (hunk 1 above)
Comment (Zahra120) [reply index.js]: to change the type of body response.

Comment (ghost) [reply index.js]: to tell the server which kind of data is coming back

Comment (izelnakri) [reply index.js]: @alivelore correct!

Comment (ghost) [reply index.js]: :+1:

**Pull request merged**